pkg/resolution/resolver/git/resolver.go

```diff
@@ -217,25 +266,155 @@ func (r *Resolver) isDisabled(ctx context.Context) bool {
 	return true
 }
 
-// ResolvedGitResource implements framework.ResolvedResource and returns
+// resolvedGitResource implements framework.ResolvedResource and returns
 // the resolved file []byte data and an annotation map for any metadata.
-type ResolvedGitResource struct {
+type resolvedGitResource struct {
 	Revision string
 	Content  []byte
+	Org      string
+	Repo     string
+	Path     string
+	URL      string
 }
 
-var _ framework.ResolvedResource = &ResolvedGitResource{}
+var _ framework.ResolvedResource = &resolvedGitResource{}
 
 // Data returns the bytes of the file resolved from git.
-func (r *ResolvedGitResource) Data() []byte {
+func (r *resolvedGitResource) Data() []byte {
 	return r.Content
 }
 
 // Annotations returns the metadata that accompanies the file fetched
 // from git.
-func (r *ResolvedGitResource) Annotations() map[string]string {
-	return map[string]string{
-		AnnotationKeyRevision:                     r.Revision,
-		resolutioncommon.AnnotationKeyContentType: YAMLContentType,
+func (r *resolvedGitResource) Annotations() map[string]string {
+	m := map[string]string{
+		internal.BuiltInAnnotation(AnnotationKeyRevision):                     r.Revision,
+		internal.BuiltInAnnotation(AnnotationKeyPath):                         r.Path,
+		internal.BuiltInAnnotation(resolutioncommon.AnnotationKeyContentType): yamlContentType,
+	}
+
+	if r.Org != "" {
+		m[internal.BuiltInAnnotation(AnnotationKeyOrg)] = r.Org
 	}
+	if r.Repo != "" {
+		m[internal.BuiltInAnnotation(AnnotationKeyRepo)] = r.Repo
+	}
+	if r.URL != "" {
+		m[internal.BuiltInAnnotation(AnnotationKeyURL)] = r.URL
+	}
+
+	return m
+}
```

Comment:
Is it possible to explicitly set the annotation keys to be namespaced directly in `annotations.go` rather than doing it at time of writing the annotations i.e. `AnnotationKeyPath=internal.BuiltInAnnotation("path")`? Doing it here implicitly might mislead that the annotation key is just original `AnnotationKeyPath` rather than the name-spaced.

Reply:
Yeah, that could be done, I just got tired of copy-pasting `resolution.tekton.dev/` everywhere. =)

Reply:
...and I'm gonna do it. First a PR to update all the existing resolvers other than `git` (since that'll be dealt with in this PR), then updating this PR.

Reply:
Oh, wait, I added it in this PR in the first place! 🤦

Reply:
aaaand done.